engine/src/main/java/org/terasology/rendering/nui/widgets/UIDropdown.java

```diff
@@ -110,7 +109,8 @@ public void onDraw(Canvas canvas) {
             for (int i = 0; i < optionListeners.size(); ++i) {
                 if (optionListeners.get(i).isMouseOver()) {
                     canvas.setMode(HOVER_MODE);
-                } else if (i==highlighted && TabbingManager.focusedWidget != null && TabbingManager.focusedWidget.equals(this)) {
+                    highlighted = i;
+                } else if (i==highlighted) {
```

Comment:
Minor whitespace issue here, could you grab a Checkstyle IDE plugin to validate that your changes follow our code guidelines please? :-)

Trivial but good to keep clean!